Add serialisation output examples

socketpair opened the pull request.

socketpair commented:
That information helped me a lot to choose correct constants. Sure, it will help other people.

reaperhulk commented on docs/hazmat/primitives/asymmetric/serialization.rst:
```diff
@@ -418,12 +418,24 @@ Serialization Formats
         Frequently known as PKCS#1 format. Still a widely used format, but
         generally considered legacy.
 
+        In PEM encoding (for RSA key) it looks like::
```
Let's say "A PEM encoded RSA key will look like"

reaperhulk commented on docs/hazmat/primitives/asymmetric/serialization.rst:
```diff
@@ -418,12 +418,24 @@ Serialization Formats
         encryption. Choose this unless you have explicit legacy compatibility
         requirements.
 
+        In PEM encoding it looks like::
```
"A PEM encoded key will look like"

reaperhulk commented on docs/hazmat/primitives/asymmetric/serialization.rst:
```diff
@@ -443,11 +455,23 @@ Serialization Formats
         identifier and the public key as a bit string. Choose this unless
         you have specific needs.
 
+        In PEM encoding it looks like::
```
"A PEM encoded key will look like"

reaperhulk commented on docs/hazmat/primitives/asymmetric/serialization.rst:
```diff
@@ -443,11 +455,23 @@ Serialization Formats
         Just the public key elements (without the algorithm identifier). This
         format is RSA only, but is used by some older systems.
 
+        In PEM encoding it looks like::
```
"A PEM encoded key will look like"

socketpair commented:
Fixed all these strings, rebased. Now it looks like:

![default](https://user-images.githubusercontent.com/1157412/41864911-e75a7dd2-78c4-11e8-96b9-10cee4de389d.png)

socketpair commented:
@reaperhulk Please review

socketpair commented:
@reaperhulk Could you please merge this PR ? checks fails caoused not by me.

socketpair commented:
@alex @reaperhulk 

I'm triggering you for merging this PR.
